tekton/ci/interceptors/github/config/rbac.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Does the interceptor actually need all these roles?

Reply:
Removed. The interceptor wasn't using any of these - I needed them to set up the trigger EL for testing.